### PR TITLE
feat(clerk-js): Add impersonation badge in active devices where appllicable

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Add an Impersonator badge (similar to the "this device" badge) next to the device if the session has an actor. 

Screenshot 2022-09-28 at 23 29 50

### CODE REVIEW

**Review by raptisj — APPROVED** [review]

File: packages/clerk-js/src/ui/localization/defaultEnglishResource.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
   footerActionLink__useAnotherMethod: 'Use another method',
   badge__primary: 'Primary',
   badge__thisDevice: 'This device',
+  badge__impersonationDevice: 'Impersonator',
```
Comment (SokratisVidros) [inline packages/clerk-js/src/ui/localization/defaultEnglishResource.ts:50]: 🔧 Can we find a better naming for this as this sounds a bit mysterious?

Comment (desiprisg) [reply packages/clerk-js/src/ui/localization/defaultEnglishResource.ts:50]: Do you mean the key or the value?

Comment (SokratisVidros) [reply packages/clerk-js/src/ui/localization/defaultEnglishResource.ts:50]: The value. Maybe:impersonateed session

Comment (desiprisg) [reply packages/clerk-js/src/ui/localization/defaultEnglishResource.ts]: Resolved via meeting

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**